Source/Classes/OverlayContainerViewController.swift

```diff
@@ -304,11 +312,13 @@ open class OverlayContainerViewController: UIViewController {
             $0.view.topAnchor.constraint(equalTo: overlayContainerView.topAnchor).isActive = true
             $0.view.bottomAnchor.constraint(equalTo: overlayContainerView.bottomAnchor).isActive = true
             $0.view.centerXAnchor.constraint(equalTo: overlayContainerView.centerXAnchor).isActive = true
-            let widthConstraint = $0.view.widthAnchor.constraint(equalTo: overlayContainerView.widthAnchor)
-            widthConstraint.priority = .defaultHigh
-            widthConstraint.isActive = true
-            widthLimitConstraint = $0.view.widthAnchor.constraint(lessThanOrEqualToConstant: 375)
-            widthLimitConstraint?.isActive = limitWidthInCompactHeight && traitCollection.verticalSizeClass == .compact
+            overlayContainerLandscapeWidthConstraint = $0.view.widthAnchor.constraint(equalToConstant: 375)
+            overlayContainerPortraitWidthConstraint = $0.view.widthAnchor.constraint(equalTo: overlayContainerView.widthAnchor)
+            if limitWidthInLandscape && traitCollection.verticalSizeClass == .compact {
+                overlayContainerLandscapeWidthConstraint?.isActive = true
+            } else {
+                overlayContainerPortraitWidthConstraint?.isActive = true
+            }
```

Comment:
@thomasdelgado have you double-checked the result using this on iPad? More just a courteous question :)

Reply:
@adamrushy it's always good to double checked! hahaha 
I didn't, but this is not being used anymore on iPad 😅